scripts/render-template.py

```diff
@@ -26,6 +26,10 @@ def _urljoin(base, url):
 
 def template_string(string, variables, templates_path):
     jinja_env = jinja2.Environment(
+        autoescape=jinja2.select_autoescape(
+            enabled_extensions=('j2'),
```

Comment:
Need an extra comma to make sure this is a tuple

```suggestion
            enabled_extensions=('j2',),
```